libkernelflinger/lib.c

```diff
@@ -1298,6 +1298,10 @@ VOID pause(UINTN seconds)
         uefi_call_wrapper(BS->Stall, 1, seconds * 1000000);
 }
 
+VOID pause_us(UINTN microseconds)
+{
+        uefi_call_wrapper(BS->Stall, 1, microseconds);
```

Comment:
It is always return true. NO delay here

Reply:
If we are using EDK2, Stall function is supported.
However, uefiwrappe does not support bs_stall actually, see implementation here:
http://inlubt0232.iind.intel.com:8080/source/xref/CELADON_S_MR0_MASTER/hardware/intel/efiwrapper/libefiwrapper/bs.c?r=79fb203a#245
So we need to implement pause ourselves, perhaps you can use 8253/8254 ?